src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Link } from "react-router-dom";
```

Comment:
You can import all the components from the module in this way -> import { Button, Container , ... } from 'blabla'. I recommend you do it like that :D.